.travis.yml

```diff
@@ -32,19 +32,16 @@ env:
 - COMPOSE_APP=postgres PYTHON_VERSION=3.6 POSTGRES_VERSION=10
 - COMPOSE_APP=postgres PYTHON_VERSION=3.6 POSTGRES_VERSION=11
 
-- COMPOSE_APP=postgres PYTHON_VERSION=3.7 POSTGRES_VERSION=9.4
 - COMPOSE_APP=postgres PYTHON_VERSION=3.7 POSTGRES_VERSION=9.5
 - COMPOSE_APP=postgres PYTHON_VERSION=3.7 POSTGRES_VERSION=9.6
 - COMPOSE_APP=postgres PYTHON_VERSION=3.7 POSTGRES_VERSION=10
 - COMPOSE_APP=postgres PYTHON_VERSION=3.7 POSTGRES_VERSION=11
 
 # Postgres gis:
-- COMPOSE_APP=postgres-gis PYTHON_VERSION=3.6 POSTGRES_VERSION=9.5 POSTGIS_VERSION=2.4
```

Comment:
@orf did you drop 9.5 here on purpose?

Reply:
It's confusing, but I moved it to the `Quick runs` section rather than deleting it.

Reply:
oh, yes -- sorry